piper/ratbagd.py

```diff
@@ -490,6 +490,26 @@ class RatbagdButton(_RatbagdDBus):
     TYPE_PROFILE_UP = 22
     TYPE_PROFILE_DOWN = 23
 
+    ACTION_SPECIAL_UNKNOWN = (1 << 30),
+    ACTION_SPECIAL_DOUBLECLICK = (1 << 30) + 1,
+    ACTION_SPECIAL_WHEEL_LEFT = (1 << 30) + 2,
+    ACTION_SPECIAL_WHEEL_RIGHT = (1 << 30) + 3,
+    ACTION_SPECIAL_WHEEL_UP = (1 << 30) + 4,
+    ACTION_SPECIAL_WHEEL_DOWN = (1 << 30) + 5,
+    ACTION_SPECIAL_RATCHET_MODE_SWITCH = (1 << 30) + 6,
+    ACTION_SPECIAL_RESOLUTION_CYCLE_UP = (1 << 30) + 7,
+    ACTION_SPECIAL_RESOLUTION_CYCLE_DOWN = (1 << 30) + 8,
+    ACTION_SPECIAL_RESOLUTION_UP = (1 << 30) + 9,
+    ACTION_SPECIAL_RESOLUTION_DOWN = (1 << 30) + 10,
+    ACTION_SPECIAL_RESOLUTION_ALTERNATE = (1 << 30) + 11,
+    ACTION_SPECIAL_RESOLUTION_DEFAULT = (1 << 30) + 12,
+    ACTION_SPECIAL_PROFILE_CYCLE_UP = (1 << 30) + 13,
+    ACTION_SPECIAL_PROFILE_CYCLE_DOWN = (1 << 30) + 14,
+    ACTION_SPECIAL_PROFILE_UP = (1 << 30) + 15,
+    ACTION_SPECIAL_PROFILE_DOWN = (1 << 30) + 16,
+    ACTION_SPECIAL_SECOND_MODE = (1 << 30) + 17,
+    ACTION_SPECIAL_BATTERY_LEVEL = (1 << 30) + 18,
+
```

Comment:
this bit has to go, it removes the fixes from benjamin's commit in libratbag, with the commas in the end the entries end up as tuples